CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Next release
 
-/
+### Added
+
+- Introduces parameters to allow the FE to tune node sizes in the network visualisation of insights.
```

Comment:
Small remark, but it's best to write your changelog entries for the end user, although it can be somewhat power user.

In this case, if it's already hooked up as well, more something like
> - When looking at a network analysis, the node size now fits the screen better

Reply:
Yes, I know. I gave it some thought but I didn't see any good solution there. I hesitated between this, writing nothing, and writing something similar to your example. But

> When looking at a network analysis, the node size now fits the screen better

would not be true since this is just the back-end part. 

What I'm sure is that I certainly don't want to introduce extra-coordination / constraints between the FE and the BE to polish changelog messages. I was also wondering if we were not conflating release notes and changelog.

Reply:
@kogre any thoughts on this? (I'm never sure whether github sends notification for replies on comment.)

Reply:
Yes, they do, I was just slow :)

I think not having it in the changelog is fine, in this case. When the FE implements it, they'll add it.